### PR TITLE
Update plugwise docs for added select platform

### PR DESCRIPTION
**Pull request by bouwew**
## Proposed change

Update plugwise docs for added select platform and various other updates/improvements.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68303
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by CoMPaTech — DISMISSED** [review]

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -42,23 +45,23 @@ The password can be found on the bottom of your Smile or Stretch, the ID, it sho
...
+Please note: when you have an Anna and an Adam, make sure to only configure the Adam integration. You can press the "IGNORE" button on the Anna integration to remove this integration. In case you need to rediscover the Anna integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".
 
-<div class='note warning'>
-When you have an Anna and an Adam, make sure to only configure the Adam integration. You can press the "IGNORE" button on the Anna integration to remove this integration. In case you need to rediscover the Adam integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".
-</div>
+For a thermostat, the active schedule can be deactivated or reactivated via the climate card. Please note, when no schedule is active, one must first be activated in the Plugwise App. Once that has been done the Plugwise Integration can manage the future operations.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown]:
```suggestion
For a thermostat, the active schedule can be deactivated or reactivated via the climate card. Please note, that when no schedule is active, one must first be activated in the Plugwise App. Once that has been done the Plugwise Integration can manage future operations.
```

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -42,23 +45,23 @@ The password can be found on the bottom of your Smile or Stretch, the ID, it sho
...
+
+Auto means the schedule is active, Heat means it's not active. The active thermostat schedule can be changed via the connected thermostat select-entity. Please note: only schedules that have two or more schedule-points will be shown as select-options.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown]:
```suggestion
Auto means the schedule is active, Heat means it's not active. The active thermostat schedule can be changed via the connected thermostat select-entity. Please note: that only schedules that have two or more schedule points will be shown as select options.
```

File: source/_integrations/plugwise.markdown [hunk 3]
```diff
@@ -76,19 +79,19 @@ script:
           entity_id: climate.anna
 ```
 
-#### Set HVAC mode (schedule)
+#### Set HVAC mode (limited to schedule active / not active)
 
 Service: `climate.set_hvac_mode`
 
-Available options include `auto` or `off`. The meaning of `auto` is that a schedule is active and the thermostat will change presets accordingly. The meaning of `off` is that there is no schedule active, i.e., the active preset or manually set temperature is to be used to control the climate of your house or rooms.
+Available options include `auto`, `heat` and `cool` (only when there is a cooling option available). The meaning of `auto` is that a schedule is active and the thermostat will change presets accordingly. The meaning of `heat/cool` is that there is no schedule active, i.e., the active preset or manually set temperature is used to control the climate in your house or rooms.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown]:
```suggestion
Available options include `auto`, `heat`, and `cool` (only when there is a cooling option available). The meaning of `auto` is that a schedule is active and the thermostat will change presets accordingly. The meaning of `heat/cool` is that there is no schedule active, i.e., the active preset or manually set temperature is used to control the climate in your house or rooms.
```

**Review by frenck — APPROVED** [review]

**Comment by bouwew** [comment]
@frenck Thanks for reviewing and approving!

**Pull request merged**